test/FunctionalTests/Microsoft.OData.Core.Tests/ODataServiceCollectionExtensionTests.cs

```diff
@@ -0,0 +1,230 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="BufferUtilsTests.cs" company="Microsoft">

```

Comment:
Looks like I didn't catch this. Should be `ODataServiceCollectionExtensionsTests.cs`. Oops.